hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -668,13 +668,21 @@ public void createBucket(
       builder.setDefaultReplicationConfig(defaultReplicationConfig);
     }
 
+    String replicationType = defaultReplicationConfig == null 
+        ? "with server-side default replication type"
+        : defaultReplicationConfig.getType().toString();
+
     String layoutMsg = bucketLayout != null
         ? "with bucket layout " + bucketLayout
         : "with server-side default bucket layout";
     LOG.info("Creating Bucket: {}/{}, {}, {} as owner, Versioning {}, " +
-            "Storage Type set to {} and Encryption set to {} ",
+            "Storage Type set to {} and Encryption set to {}, " +
+            "Replication Type set to {}, Namespace Quota set to {}, " + 
```

Comment:
Can you please tweak the replication-related part a bit?

1. `Replication Type set to with server-side default replication type` sounds awkward.
2. I think it should include not only replication type, but all details of the replication config, available via `configFormat()` of `ReplicationConfig`.  But I just realized `DefaultReplicationConfig` does not easily provide those details, so I created a PR (#4005) to expose them.  If that's merged, we can make the message more helpful.

Reply:
Thanks for your comments @adoroszlai , I will modify this after you finish your changes

Reply:
Finally #4005 is merged, sorry for taking so long.

Reply:
Hey @chrisGe123 ! Just a heads-up that Attila has completed his changes~

Reply:
thanks, @DaveTeng0 , I have finished the code changes but have some issues with my account of github, will submit this commit today or tomorrow.

Reply:
@DaveTeng0 and @adoroszlai I am still waiting for Sammi to come back from holidays to help me out about github issue, but you guys can help me to review the test result as followed to see if this message change is ok:

```
# bin/ozone sh bucket create /vol1/bucket2 --layout FILE_SYSTEM_OPTIMIZED --type EC --replication rs-3-2-1024k
23/01/19 07:53:18 INFO rpc.RpcClient: Creating Bucket: vol1/bucket2, with bucket layout FILE_SYSTEM_OPTIMIZED, root as owner, Versioning false, Storage Type set to DISK and Encryption set to false, Replication Config is set to EC/ECReplicationConfig{data=3, parity=2, ecChunkSize=1048576, codec=rs}, Counts Quota set to -1, Space Quota set to -1 bytes


# bin/ozone sh bucket create /vol1/bucket3
23/01/19 07:54:33 INFO rpc.RpcClient: Creating Bucket: vol1/bucket3, with server-side default bucket layout, root as owner, Versioning false, Storage Type set to DISK and Encryption set to false, Replication Config is set to server-side default replication config, Counts Quota set to -1, Space Quota set to -1 bytes


# bin/ozone sh bucket create /vol1/bucket3 --layout OBJECT_STORE --replication-type EC --replication rs-6-3-1024k --namespace-quota=10 --space-quota=1073741824
23/01/19 07:55:27 INFO rpc.RpcClient: Creating Bucket: vol1/bucket3, with bucket layout OBJECT_STORE, root as owner, Versioning false, Storage Type set to DISK and Encryption set to false, Replication Config is set to EC/ECReplicationConfig{data=6, parity=3, ecChunkSize=1048576, codec=rs}, Counts Quota set to 10, Space Quota set to 1073741824 bytes
BUCKET_ALREADY_EXISTS Bucket already exist

# bin/ozone sh bucket create /vol1/bucket4 --layout OBJECT_STORE --replication-type EC --replication rs-6-3-1024k --namespace-quota=10 --space-quota=1073741824
23/01/19 07:55:48 INFO rpc.RpcClient: Creating Bucket: vol1/bucket4, with bucket layout OBJECT_STORE, root as owner, Versioning false, Storage Type set to DISK and Encryption set to false, Replication Config is set to EC/ECReplicationConfig{data=6, parity=3, ecChunkSize=1048576, codec=rs}, Counts Quota set to 10, Space Quota set to 1073741824 bytes
```